recap-of-techtalks-summer-edition-2021.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Recap of our TechTalks summer edition 2021
+date: 2021-11-23
+author: "federicab"
+category: "techtalks"
+tags:
+  - post
+  - guest-post
+layout: article
+metaIMG: "womendotcodebe-techtalks-recap-june2021.jpg"
+featured: true
+---
+
+Buckle up for the recap of another eclectic Techtalk.
+Lisa Tarpletti shared some tip on how to eco-design our system, Asma Oualmakran shed light on cybersecurity buzzwords, and Liza Morrison is walked us through a brand-new framework, Qwick.
+
+<!-- excerpt -->
+
+## Lisa Trapletti - Eco-designing its digital services. A path to sustainable IT
+
+{% image {src: "./blog/guest-blogs/images/techtalks-sophie.png", alt: "Sophie Ragas", widths: [300]} %}
+
+Lisa is a senior software developer working for iCure. She is certified in eco-design of digital services and passionate about the environmental transition and digital transformation.
+
+### The impact of IT
+
+Let’s kick off with some shocking numbers on the IT industry.
+
+Computing technologies are part of the daily life of 5 billions of users with 34 billions of connected devices, and it’s estimated that these numbers will double by 2025! 😱
+
+Also the environmental costs of IT’s ever-expanding presence are skyrocketing. Here are some figures:
+
+- The IT industry produces 4% of the greenhouse gas emissions in the world (more than the aviation sector which is responsible only for 2% of emissions)
+- It takes 10% of the world-wide electricity consumption
+- It is responsible for abiotic resources depletion
+- It perpetrates inhumane working conditions. In the Democratic Republic of Congo 6 million of workers died in coltan mines, a material used to produce electronics.
+
+Let’s break the IT industry into its component to understand why it has such a big ecological impact.There are three main actors in IT:
+
+1. The end user device
+2. The network infrastructure
+3. The data centres
+
+We can identify two sources of impact:
+
+1. The manufacture of equipments
+2. The manufacture of electricity
```

Comment:
Not sure if manufacture is the right word here.

The process of generating electricity?